e2e/features/step_definitions/amend-elements-issues.steps.ts

```diff
@@ -0,0 +1,45 @@
+import { When, Then } from 'cucumber';
+import { AnyCcdFormPage } from '../../pages/any-ccd-form.page';
+
+import { browser } from 'protractor';
+import { expect } from 'chai';
+
+const anyCcdPage = new AnyCcdFormPage();
+
+
+When('I select {string} and {string} Elements', async function (string, string2) {
+    await anyCcdPage.clickElementById('elementsDisputedList-housing');
+    await anyCcdPage.clickElementById('elementsDisputedList-childcare');
+    await anyCcdPage.click('Continue');
+  });
+
+When('I add issue codes for respective elements', async function () {
+    await browser.sleep(3000);
+    expect(await anyCcdPage.pageHeadingContains('Issue codes')).to.equal(true);
+    await anyCcdPage.addNewCollectionItem('Housing');
+    await anyCcdPage.selectHousingIssueCode();
+    await browser.sleep(3000);
+    await anyCcdPage.addNewCollectionItem('Childcare');
+    await anyCcdPage.selectChildcareIssueCode();
+    await browser.sleep(500);
+    await anyCcdPage.click('Continue');
+});
+
+When('I submit the event', async function () {
```

Comment:
We do have below existing method to perform submit action within events -
```
When(/^I submit "(.+)"$/, async function (action) {
    // await anyCcdPage.click('Continue');
    await browser.sleep(4000);
    expect(await anyCcdPage.pageHeadingContains(action)).to.equal(true);
    await anyCcdPage.click('Submit');
    await browser.sleep(1000);
});
```
So in the feature file we just need to change the step to below : -
`When I submit "Amend elements"`

And can delete method on line 28

Reply:
Fixed